qt/draw_widget.cpp

```diff
@@ -672,6 +672,12 @@ void DrawWidget::ShowPlacePage()
       LOG(LERROR, ("Error while trying to edit feature."));
     }
   }
+
+  if (auto routePointAddMode = std::move(dlg.GetRoutePointAddMode()))
```

Comment:
Why std::move is needed?

Reply:
In order to avoid unnecessary copying.

Reply:
Can you please clarify which copying are you avoiding using std::move here?

Reply:
Upon inspecting the generated code of some typical compilers, it turns out that this move was counter productive, so it has been removed.